commands/trade.js

```diff
@@ -7,10 +7,16 @@ const async = require('async');
  * @example   !tr[ade] <item name>
  */
 exports.run = async (client, message, args, level) => { // eslint-disable-line no-unused-vars
-  const item_list_cache = client.tornData.items;
+  let item_list_cache = client.tornData.items;
```

Comment:
I think this should still be a const because you are not changing the reference, only a value inside the object.